pkg/operator/starter.go

```diff
@@ -266,7 +267,7 @@ func RunOperator(ctx context.Context, controllerContext *controllercmd.Controlle
 	)
 
 	quorumChecker := ceohelpers.NewQuorumChecker(
-		kubeInformersForNamespaces.InformersFor("").Core().V1().Namespaces().Lister(),
+		clusterInformers.InformersFor("").Core().V1().Namespaces().Lister(),
```

Comment:
quorumChecker needs to list all namespaces?

Reply:
fixed to only use a static informer/lister with one namespace by field selector